### PR TITLE
Removed artwork auction results

### PR DESCRIPTION
**Pull request by sarahscott**
✨

### CODE REVIEW

File: Artsy/View_Controllers/Artwork/ARArtworkViewController+ButtonActions.m [hunk 1]
```diff
@@ -149,7 +147,7 @@ - (void)bidCompleted:(SaleArtwork *)saleArtwork
 
     ARAuctionWebViewController *viewController = [ARSwitchBoard.sharedInstance loadBidUIForArtwork:self.artwork.artworkID
                                                                                             inSale:saleArtwork.auction.saleID];
-    [self.navigationController pushViewController:viewController animated:ARPerformWorkAsynchronously];
+    [self.navigationController pushViewController:(UIViewController *)viewController animated:ARPerformWorkAsynchronously];
```
Comment (alloy) [inline Artsy/View_Controllers/Artwork/ARArtworkViewController+ButtonActions.m]: Weird this, must be because you removed the import for this VC no?

**Comment by alloy** [comment]
405 deletions :metal:

**Comment by alloy** [comment]
Feel free to merge after fixing that import/cast.

File: Artsy/View_Controllers/Artwork/ARArtworkViewController+ButtonActions.m (hunk 1 above)
Comment (sarahscott) [inline Artsy/View_Controllers/Artwork/ARArtworkViewController+ButtonActions.m]: Oh woops

Comment (sarahscott) [inline Artsy/View_Controllers/Artwork/ARArtworkViewController+ButtonActions.m]: Good c

**Comment by ArtsyMobile** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%">1 Warning</th>
     </tr>
  </thead>
  <tbody>
        <tr>
      <td>:warning:</td>
      <td>Needs testing on a Phone if change is non-trivial</td>
    </tr>
    
  </tbody>
</table>
  

  


<p align="right" data-meta="generated_by_danger" data-base-commit="" data-head-commit="" >
  Generated by :no_entry_sign: <a href="https://github.com/danger/danger/">danger</a>
</p>

**Pull request merged**